examples/plugins/cmd-runner/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -x
+
+# This is the entrypoint for the image and meant to wrap the
+# logic of gathering/reporting results to the Sonobuoy worker.
+
+results_dir="${RESULTS_DIR:-/tmp/results}"
+
+if [[ $# -eq 0 ]]; then
+	echo "No arguments; expects each argument to be a shell command."
```

Comment:
If the user does this and doesn't pass args, what happens in Sonobuoy? Just curious the workflow to identify errors and what folks would run into.

Reply:
Thanks, good catch. This is actually the exact problem that the upstream k8s e2e tests had. It errored out and never told the worker it was done so Sonobuoy hangs. Will fix.

Reply:
Uses a `trap ... EXIT` now. When no arguments this error is dumped to `results_dir/out` just for consistent location/naming.